src/main/java/com/umbr3114/models/CommentModel.java

```diff
@@ -0,0 +1,59 @@
+package com.umbr3114.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.bson.types.ObjectId;
+import org.mongojack.MongoCollection;
+
+import java.text.DateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.text.SimpleDateFormat;
+
+@MongoCollection(name = "comments")
+public class CommentModel {
+    public ObjectId _id;
+    public String bodyText;
+    public String authorId;
+    @JsonProperty("post_id")
+    public String postId;
+    @JsonProperty("date")
+    public Date date;
+    public String dateString;
+
+    public String getBodyText() {
+        return bodyText;
+    }
+
+    public void setBodyText(String bodyText) {
+        this.bodyText = bodyText;
+    }
+
+    public String getAuthorId() {
+        return authorId;
+    }
+
+    public Date getDate() { return date; }
+
+    public String getDateString() { return dateString; }
+
+    public void setDate() { this.date = new java.util.Date(System.currentTimeMillis());
```

Comment:
this setter should not be creating the date, but taking in a date as a parameter. since date is a public variable in the ModelClass, a setter is also not required, since anybody has access to the variable.